src/components/pages/Offers/Offers.jsx

```diff
@@ -55,12 +51,9 @@ class Offers extends PureComponent {
       selectedVenueId: selectedVenueId || ALL_VENUES,
       selectedTypeId: selectedTypeId || ALL_TYPES,
       venueOptions: [],
-      statusFilters: {
-        active: !isFilteredByActiveStatus,
-        inactive: !isFilteredByInactiveStatus,
-      },
+      status: mapBrowserToApi[status] || ALL_STATUS,
+      isFilteredByStatus: !!status,
```

Comment:
```suggestion
      isFilteredByStatus: !!status,
```
Double not donc rien du tout non ?

Reply:
Hum, c'est vrai que ce n'est clairement pas explicite. En fait c'est une manière de passer une valeur en booléen (ici `status` est une string). Je peux changer pour `Boolean(status)` pour plus de clarté.